loops.js

```diff
@@ -5,20 +5,28 @@
 // Hoera!
 // Hoera!
 // ==========================================
-
-
+for (i=0; i<3; i++) {
+    console.log("Hoera")
+}
+console.log("\n"); //print lege ruimte tussen uitkomsten in de console
 
 // ==========================================
 // Opdracht 2. Maak een for-loop die 4 keer het woord "loop..." logt, en bij de vijfde loop het woord "klaar!"
 // Verwachte uitkomsten:
-// loop...
+// loop.
 // loop...
 // loop...
 // loop...
 // klaar!
  // ==========================================
-
-
+for (i=0; i<5; i++) {
+    if (i==4) {
+        console.log("Klaar");
+        break;
```

Comment:
de break hier werkt wel, want de uitkomst is hetzelfde als dat er hier en else zou staan. Maar met de break gaat hij uit de loop en als je nog verder had gewild dan mis je dit. Met else maakt het je if structuur net wat gecontroleerder.